index-el.html

```diff
@@ -56,7 +56,7 @@ <h5>
 
     <p>
       <a href="https://www.preprints.org/manuscript/202004.0203/v1">Τώρα</a> <a href="https://link.springer.com/article/10.1007/s11606-020-06067-8">γνωρίζουμε</a> <a href="https://science.sciencemag.org/content/368/6498/1422">ότι</a> οι μάσκες έχουν μια σημαντική επίδραση στην επιβράδυνση της μετάδοσης του κορονοϊού.
-      Παρά τα στοιχεία, κάποιοι άνθρωποι διαφωνούν με τη χρήση τους, διότι θεωρούν ότι είναι μια προσωπική επιλογή και όχι ένα θέμα για την προστασία της δημόσιας υγείας.
+      Παρά τα στοιχεία, κάποιοι άνθρωποι διαφωνούν με τη χρήση τους, διότι θεωρούν ότι είναι μια προσωπική επιλογή και όχι ένα δημόσιο θέμα για την προστασία της υγείας όλων μας.
     </p>
 
     <p>
```

Comment:
'για να σταματήσουμε μια επιδημια' ---> 'για να σταμαγτήσουμε μια πανδημία'

Reply:
@ismarou This change is best to be proposed to the original english text, which uses only the term "epidemic". If you feel that's important I'd think it's best to open an issue on the English text. We'll translate the original as is then.

Reply:
I get what you say. Opening the issue in the English text would cause the extra problem/need of adapting all other translations, previous to ours, so I don't know how viable it really is. Since you have done all the work, it's your call to choose if you wish to be exact in terms of translation or semantics.